tests/token.test.ts

```diff
@@ -86,4 +86,18 @@ describe('token', () => {
     const isValid = await token.isValid(childUcan)
     expect(isValid).toBe(false)
   })
+
+  it('identifies a ucan that is not active yet', async () => {
+    const badUcan = {
+      ...ucan,
+      payload: {
+        ...ucan.payload,
+        nbf: 2637252774,
+        exp: 2637352774
+      }
+    }
+
+    const isTooEarly = await token.isTooEarly(badUcan)
+    expect(isTooEarly).toBe(true)
+  })
```

Comment:
Minor request: can we also test the counterfactual? (i.e. where we have a valid UCAN, this is `false`)

Reply:
Good idea! I've added a counterfactual test case.